src/components/gas/GasSpeedButton.tsx

```diff
@@ -348,21 +346,13 @@ const GasSpeedButton = ({
 
   const openGasHelper = useCallback(async () => {
     Keyboard.dismiss();
-    if (crossChainServiceTime) {
```

Comment:
also leaving a note here for confirming explanation of removal of `crossChainServiceTime`

Reply:
Same explanation here